docs/stylesheets/custom-style.css

```diff
@@ -446,3 +446,19 @@
 .md-typeset code {
   word-break: normal;
 }
+
+#myModal {
```

Comment:
rename to sth more descriptive -> cookieModal and cookiePopup

Reply:
I see that you already have classes on the html elements, ca-accept-cookies and popuptext. Use these classNames instead and remove the id. And you might rename ca-accept-cookies to just accept-cookies since ca is abbreviation for custom analytics which is our first example site